### PR TITLE
Fix rwd header buttons

### PR DESCRIPTION
**Pull request by nico3333fr**
- reordered buttons/title
- added border on other steps

image

### CODE REVIEW

File: src/app/containers/SignupContainer/SignupContainer.js [hunk 1]
```diff
@@ -124,16 +124,16 @@ const SignupContainer = ({ history, onLogin }) => {
...
-                <div className="flex flex-nowrap mb1">
-                    <div className="flex-item-fluid">
+                <div className="flex flex-nowrap onmobile-flex-wrap mb1">
+                    <div className="flex-item-fluid plan-back-button">
                         <Button onClick={handleBackClick}>
                             {prevStep ? c('Action').t`Back` : c('Action').t`Homepage`}
                         </Button>
                     </div>
-                    <div className="w">
+                    <div class="onmobile-min-w100 onmobile-aligncenter onmobile-mt0-5">
```
Comment (EpokK) [inline src/app/containers/SignupContainer/SignupContainer.js]:
```suggestion
                    <div className="onmobile-min-w100 onmobile-aligncenter onmobile-mt0-5">
```

File: src/app/containers/SignupContainer/AccountStep/AccountStep.js [hunk 2]
```diff
@@ -19,13 +19,13 @@ const AccountStep = ({ onContinue, model, children }) => {
     };
 
     return (
-        <>
+        <div className=" border-top pt3 mb2">
```
Comment (EpokK) [inline src/app/containers/SignupContainer/AccountStep/AccountStep.js]:
```suggestion
        <div className="border-top pt3 mb2">
```

File: src/app/containers/SignupContainer/PaymentStep/PaymentStep.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ const PaymentStep = ({ onPay, paymentAmount, model, children }) => {
     const handlePayment = () => withLoading(onPay(model, parameters));
 
     return (
-        <>
+        <div className=" border-top pt3 mb2">
```
Comment (EpokK) [inline src/app/containers/SignupContainer/PaymentStep/PaymentStep.js]:
```suggestion
        <div className="border-top pt3 mb2">
```

File: src/app/containers/SignupContainer/SignupContainer.js (hunk 1 above)
Comment (nico3333fr) [reply src/app/containers/SignupContainer/SignupContainer.js]: Oops, done via https://github.com/ProtonMail/proton-vpn-settings/pull/175/commits/b39d935dd98bd393399f5bf0e0be3326676cde06

**Pull request merged**